### ISSUE TITLE
Django CMS front-end text plugin editor modal is unusably short in Chrome

### ISSUE THREAD

**Issue by ghing** [denverfoundation/storybase#82]
When I try to edit a text plugin in Chrome, the modal pops up, but is too short:

![Plugin editor modal is too short in Chrome ...](http://i.imgur.com/oldC0.png)

It works just fine in Firefox:

![But it's fine in Firefox ...](http://i.imgur.com/rBj8Q.png)

In Chrome, TinyMCE and the form elements all display, they just aren't visible because of short height of the modal element.

I'm using django-cms 2.2 and Chrome Version 19.0.1084.30 beta on Ubuntu (though others have reported seeing this in Chrome on other platforms)

I've opened an upstream ticket with the Django CMS developers at https://github.com/divio/django-cms/issues/1215

**Comment by ghing** [comment]
cc @jwirfs-brock 

I've opened an upstream ticket for this. If it doesn't get much activity, I bet I can fix it myself, but I'll hold off on that at the moment.

**Comment by ghing** [comment]
I did a little more poking around with this and it works correctly on the development instance.  I think the issue is caused by the fact that on staging javascript for tinymce is served from a separate domain.

In Chrome, this is the error that is reported:

```
Unsafe JavaScript attempt to access frame with URL http://staging.denverchildrenscorridor.org/admin/cms/page/2/edit-plugin/2/?popup=true&no_preview from frame with URL http://staging.denverchildrenscorridor.org/?edit. Domains, protocols and ports must match.
 jquery.min.js:99
c.each.contents jquery.min.js:99
c.extend.map jquery.min.js:31
c.each.c.fn.(anonymous function) jquery.min.js:99
(anonymous function) cms.placeholders.js:134
```

The next bit of testing that needs to go into this is to figure out if this still occurs if TinyMCE is turned off.

**Comment by ghing** [comment]
This error doesn't occur when TinyMCE is enabled (though the default editor for Django CMS, Wymeditor, throws its own set of cross-domain errors).  So, this is tied to the TinyMCE iframe, I think, and the errors are just getting propogated up to the top-level iframe (which is on the same domain and should be ok).

**Comment by ghing** [comment]
The workaround to this is to just serve TinyMCE from the application server instead of the static server.  I think this is okay as the number of users using the Django admin will be limited. I don't think we'll use TinyMCE for end-user interfaces, or if we do, we can serve it separately.

To do this, I copied the tiny_mce distribution directory to media/js/tiny_mce

Then, I edited the settings file to have the following configuration:

```
...

TINYMCE_JS_URL = MEDIA_URL + 'js/tiny_mce/tiny_mce.js'
TINYMCE_JS_ROOT = MEDIA_ROOT + 'js/tiny_mce'

...
```
